Gemfile

```diff
@@ -45,7 +45,7 @@ gem 'tracker_api',    '~> 1.6'
 gem 'travis',         '~> 1.7.6'
 
 gem 'awesome_spawn',        '>= 1.4.1'
-gem 'default_value_for'
+gem 'default_value_for',    '>= 3.1.0'
```

Comment:
I updated `default_value_for` to the latest version too. I can remove this if you don't think it's needed. /cc @Fryguy